### DISCUSSION TITLE
posts/2022/07/really-getting-started-hugo-four-steps/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#44] [Announcements]
# posts/2022/07/really-getting-started-hugo-four-steps/

Here is just the actual procedure from yesterday’s post about making things easier for new Hugo users.

https://www.brycewray.com/posts/2022/07/really-getting-started-hugo-four-steps/

**Comment by juggledad** [comment]
You need to update the thread due to the changes in HUGO. Currently when you run 'hugo new site my-site' there are different names and new folders created so you end up with:
 
my-site   <-- The Hugo project folder
└─ archetypes
└─ assets
└─ content
└─ data
└─ hugo.toml   <-- The only non-folder
└─ i18n
└─ layouts
└─ public
└─ static
└─ themes

Once you adapt to these changes, the site does come up. Thanks for the post!

**Reply by brycewray** [reply]
Noted, although I believe it still respects the name `config.toml` to avoid breaking all those sites still using older versions. Thanks for the reminder!

**Edit**: Now fixed in this post and its [follow-up](https://www.brycewray.com/posts/2022/07/really-getting-started-hugo-four-steps/).

**Comment by DF4OR** [comment]
Hi Bryce.
Yourapproachis somuch moreaccessible for me, many thanks. I _do_ have a hugo driven website up and running for a while, but each time I try to modify the looks I get lost. Youzr simple approach really helped me understand the inner workings. Thanks.

The Hugo docs are huge, but sometimes confusing and leading me in circular ways when trying to grasp the concept.

**Reply by brycewray** [reply]
@DF4OR Glad to be of help! Understanding the inner workings is the whole idea, indeed.
